components/dashboard/ExpenseReporter/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { gql, useQuery } from '@apollo/client'
+import React from 'react'
+import { Bar, BarChart, CartesianGrid, Legend, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts'
+
+import DateHelper from '../../../lib/date-helper'
+
+function ExpenseReporter() {
+  const today = new Date()
+  const lastSixMonth = DateHelper.dateBeforeSixMonths()
+
+  const urlLink = `?startmonth=${today.getFullYear()}${today.getMonth()}&endmonth=${lastSixMonth.getFullYear()}${lastSixMonth.getMonth()}`
```

Comment:
Like we discussed in the call yesterday, send the data as prop to the component and make calls to the API in the dashboard page

Reply:
Couldn't find much documentation on how to call apollo client in nextjs, but here: https://www.apollographql.com/blog/apollo-client/next-js/next-js-getting-started/
It's recommended to make graphql query calls in individual components.